<launch>
	<!-- vim: set ft=xml noet : -->
	<!-- Experimental launch script for APM based Drone -->

	<!-- Start mavros -->
	<arg name="use_sim" default="true"/>
	<include file="$(find krti2022)/launch/mavros_only.launch"/>
	<rosparam param="use_sim" subst_value="true">$(arg use_sim)</rosparam>
	
	<!-- Start gazebo -->
	<arg name="pola" default="1"/>
  	<arg name="target" default="1"/>
	<include file="$(find krti2022)/launch/gazebo_only.launch">
		<arg name="target" value="$(arg target)"/>
      	<arg name="pola" value="$(arg pola)"/>
	</include>
	<!-- <include file="$(find krti2022)/launch/vision.launch"/> -->
</launch>
